main/mafia.hs

```diff
@@ -500,14 +502,39 @@ ghciArgs extraIncludes paths = do
 
   extras <- concat <$> mapM reifyInclude extraIncludes
 
-  let dirs = ["src", "test", "gen", "dist/build/autogen"] <> extras
-  includes  <- catMaybes <$> mapM ensureDirectory dirs
+  let
+    dirs =
+      ["src", "test", "gen", "dist/build/autogen"] <> extras
```

Comment:
As per hipchat GHCi doesn't know how to rebuild these generated modules, so it won't reload the way you might expect. however right now it never works and transitively poisons all user code, so this is a far lesser evil